start.go

```diff
@@ -216,11 +215,14 @@ func start(c *cli.Context) {
 					}
 
 					// This is a proxy function, so set the http status code and return the body
-					if proxy.StatusCode != 0 {
-						w.WriteHeader(proxy.StatusCode)
+					if statusCode, err := proxy.StatusCode.Int64(); err != nil {
+						w.WriteHeader(200)
```

Comment:
You should return a BadGateway here since StatusCode is not integer. Isn't it?

Reply:
When testing the real API Gateway returned 200 if it couldn't parse any of the fields.

I think it makes sense to be more strict since we want users to catch such bugs when developing, and not when they deployed the code.

Reply:
Agreed. We should mirror API Gateway functionality.